usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -51,9 +51,13 @@
 from usaspending_api.common.exceptions import UnprocessableEntityException
 from usaspending_api.search.filters.elasticsearch.filter import _QueryType
 from usaspending_api.search.filters.time_period.decorators import NewAwardsOnlyTimePeriod
-from usaspending_api.search.filters.time_period.query_types import AwardSearchTimePeriod
+from usaspending_api.search.filters.time_period.query_types import AwardSearchTimePeriod, SubawardSearchTimePeriod
 from usaspending_api.submissions.models import SubmissionAttributes
 
+from typing import Any
```

Comment:
Can you move this towards the top? Usually we will try to format the imports in the following order:

* Python built-in
* 3rd party
* USAspending imports

Where within each group we format such that you import an entire module above the imports that are more specific. For example:

```
import os
from typing import List

import pytest

from usaspending_api... import <FUNC_NAME>
```

Reply:
I will admit it is not the most consistent so I won't block over this, but it would be good to try and be consistent.